fluss-flink/fluss-flink-common/src/main/java/com/alibaba/fluss/flink/tiering/committer/TieringCommitOperator.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (c) 2025 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.fluss.flink.tiering.committer;
+
+import com.alibaba.fluss.config.Configuration;
+import com.alibaba.fluss.flink.tiering.source.TableBucketWriteResult;
+import com.alibaba.fluss.lakehouse.committer.LakeCommitter;
+import com.alibaba.fluss.lakehouse.writer.LakeTieringFactory;
+import com.alibaba.fluss.metadata.TableBucket;
+import com.alibaba.fluss.metadata.TablePath;
+
+import org.apache.flink.streaming.api.operators.AbstractStreamOperator;
+import org.apache.flink.streaming.api.operators.OneInputStreamOperator;
+import org.apache.flink.streaming.api.operators.StreamOperatorParameters;
+import org.apache.flink.streaming.runtime.streamrecord.StreamRecord;
+
+import javax.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static com.alibaba.fluss.utils.Preconditions.checkState;
+
+/**
+ * A Flink operator to aggregate {@link WriteResult}s by table to {@link Committable} which will
+ * then be committed to lake & Fluss cluster.
+ */
+public class TieringCommitOperator<WriteResult, Committable>
+        extends AbstractStreamOperator<CommittableMessage<Committable>>
+        implements OneInputStreamOperator<
+                TableBucketWriteResult<WriteResult>, CommittableMessage<Committable>> {
+
+    private static final long serialVersionUID = 1L;
+
+    private final LakeTieringFactory<WriteResult, Committable> lakeTieringFactory;
+    private final TableLakeSnapshotCommitter tableLakeSnapshotCommitter;
```

Comment:
The tableLake prefix confused me a little, how about  rename to `FlussLakeSnapshotCommiter` as we use `LakeCommitter<WriteResult, Committable> lakeCommitter` in next code piece?

Reply:
Good suggestion.